webview.h

```diff
@@ -1009,6 +1009,12 @@ class cocoa_wkwebview_engine {
     objc::msg_send<void>(m_webview, "initWithFrame:configuration:"_sel,
                          CGRectMake(0, 0, 0, 0), config);
     objc::msg_send<void>(m_webview, "setUIDelegate:"_sel, ui_delegate);
+    if (m_debug) {
+      objc::msg_send<void>(
```

Comment:
I believe this will crash on macOS versions earlier than 13.3 where `setInspectable` doesn't exist. Tested on macOS 12.6.6.